tests/versions/go.yml

```diff
@@ -1,16 +1,17 @@
 GO_AGENT:
   - 'github;master'
+  - 'github;v1.2.0'
 
 exclude:
-  - GO_AGENT: 'github;master'
-    APM_SERVER: '6.7'
-  - GO_AGENT: 'github;master'
-    APM_SERVER: '6.6;--release'
-  - GO_AGENT: 'github;master'
-    APM_SERVER: '6.5;--release'
   - GO_AGENT: 'github;master'
     APM_SERVER: '6.4;--release'
   - GO_AGENT: 'github;master'
     APM_SERVER: '6.3;--release'
   - GO_AGENT: 'github;master'
     APM_SERVER: '6.2;--release'
+  - GO_AGENT: 'github;v1.2.0'
+    APM_SERVER: '6.4;--release'
+  - GO_AGENT: 'github;v1.2.0'
+    APM_SERVER: '6.3;--release'
+  - GO_AGENT: 'github;v1.2.0'
+    APM_SERVER: '6.2;--release'
```

Comment:
The go agent does not support v1, so it should be completely removed from this branch.